lib/daemons/recurring_tasks.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require File.expand_path("base", File.dirname(__FILE__))
+
+Daemons::Base.new("recurring_tasks").start do
+  RecurringTaskConfig.recurring_tasks.each do |(worker, method)|
+    worker.new.public_send(method)
+  end
+
+  sleep 1.minute.to_i
```

Comment:
I thought that it was somewhat important to run as close to the top-of-the-minute as possible, for billing reasons....so, wouldn't we want this to sleep only as many seconds as needed to get to the next minute? (maybe I'm being overly complicated)

Reply:
I think this is a good idea, to be safe